proparse/src/main/java/org/prorefactor/treeparser/ParseUnit.java

```diff
@@ -22,11 +22,13 @@
 import org.prorefactor.macrolevel.IncludeRef;
 import org.prorefactor.macrolevel.MacroLevel;
 import org.prorefactor.macrolevel.MacroRef;
+import org.prorefactor.proparse.IntegerIndex;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.prorefactor.proparse.IntegerIndex'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AUselessImportCheck)